src/native/external/llvm-libunwind-version.txt

```diff
@@ -0,0 +1,5 @@
+v9.0.0
+https://github.com/llvm/llvm-project/releases/tag/llvmorg-9.0.0
+
+Apply https://github.com/dotnet/runtime/commit/371905898f2c036cc30d662e8a6df162a21967a5
+Reapply changes from https://github.com/dotnet/runtime/commit/62075f115efeecaee9d94f87d0766d657a72fa76
```

Comment:
This file keeps track of how to reproduce `src/native/external/llvm-libunwind/` directory from scratch. When we will upgrade to `https://github.com/llvm/llvm-project/releases/tag/llvmorg-14.0.6` for example, we can update this file to reflect how are we diverging from upstream. It will also help us plan upstreaming the local patches.

Reply:
I think https://github.com/dotnet/runtime/commit/62075f115efeecaee9d94f87d0766d657a72fa76 has the changes that the upstream also has.  
I am ok with including the sha of that commit, if it makes the tracking more convenient, just wonder if that is expected.

Reply:
Those changes were added in v14.0.0-rc1: 
![image](https://user-images.githubusercontent.com/3840695/179663130-338da179-33b3-4f80-8acd-e44120ff4776.png)


our current version is 9.0.0. We can adjust the these additional deltas once we upgrade to newer version.

Reply:
Ah, I see. Makes sense.